docs/using.rst

```diff
@@ -655,8 +612,106 @@ The following files are available:
 
 .. _hooks:
 
-Pre and Post Validation Hooks
-=============================
+Hooks
+-----
```

Comment:
Changing this from "=" to "-" made this a subsection of "Where are my certificates?". Is this what you wanted?